app/user/rest.py

```diff
@@ -363,7 +363,7 @@ def send_user_confirm_new_email(user_id):
         notification_type=template.template_type,
         api_key_id=None,
         key_type=KEY_TYPE_NORMAL,
-        reply_to_text=service.get_default_reply_to_email_address(),
+        reply_to_text=None,
     )
```

Comment:
Nit: Passing `reply_to_text` explicitly to persist_notification is not necessary if it's `None`

Reply:
deleted